### DISCUSSION TITLE
How to turn off word wrapping?

### DISCUSSION THREAD

**Discussion by AbraxasDMG** [filebrowser/filebrowser#3594] [Q&A]
Hi everyone.
I just installed filebrowser for a test drive. When I open a file in its editor, how can I turn off the word wrapping?
It seems that the editor used in filebrowser is ace editor. Is there a possibility to set the configuration for it in filebrowser config files or to access the ace editor's default settings panels (via F1, or ctrl+,)?

Thanks in advance!

**Comment by feklee** [comment]
I have the same question. Somehow, on my phone with touch, I managed to open the the Ace settings menu, but I cannot replicate that anymore. On the desktop, I tried *Ctrl-,* and *F1*. Both have no effect.

(I actually would be prefer a simpler text file viewer. Ace does too much. In particularly if overrides default text selection, which hampers usability.)
